transports/quic/src/lib.rs

```diff
@@ -90,10 +90,6 @@ pub enum Error {
     #[error(transparent)]
     Io(#[from] std::io::Error),
 
-    /// The task to drive a quic endpoint has crashed.
-    #[error("Endpoint driver crashed")]
-    EndpointDriverCrashed,
```

Comment:
Removing this is a breaking change. Can you just deprecate the variant to say that it is no longer emitted and we remove it in the next breaking release?

Reply:
yup, did not remember that, thanks Thomas! Addressed